### PR TITLE
III-3949 - fix creation place

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Changed

- naming of event specific things to offer

### Fixed

- Creation place

---

Ticket: https://jira.uitdatabank.be/browse/III-3949

### CODE REVIEW

File: src/hooks/api/places.ts [hunk 1]
```diff
@@ -221,14 +288,22 @@ const addPlace = async ({
...
 }: AddPlaceArguments) =>
   fetchFromApi({
-    path: `/place`,
+    path: `/places`,
     options: {
       method: 'POST',
       headers,
-      body: JSON.stringify({ calendar, address, mainLanguage, name, type }),
+      body: JSON.stringify({
+        calendar,
```
Comment (simon-debruijn) [inline src/hooks/api/places.ts:300]: I think we'll also have to refactor this like on event.

File: src/i18n/nl.json [hunk 2]
```diff
@@ -152,6 +152,11 @@
     },
     "location": {
       "title": "Waar vindt dit evenement of deze activiteit plaats?",
+      "validation_messages": {
+        "street_and_number": {
+          "required": "Je straat en nummer voldoen niet aan de vereisten."
+        }
+      },
```
Comment (simon-debruijn) [inline src/i18n/nl.json:159]: Needs to be added to `fr` and `de`

File: src/hooks/api/places.ts (hunk 1 above)
Comment (brampauwelyn) [reply src/hooks/api/places.ts:300]: Oh yes, `terms` instead `type` you mean?

Comment (simon-debruijn) [reply src/hooks/api/places.ts]: I was talking about not nesting it in calendar:
Also I noticed we were not passing openingHours anywhere

https://github.com/cultuurnet/udb3-frontend/pull/484/commits/07f58cc7c08cb9d8792d296d5772e2adb3f64e5f

**Review by simon-debruijn — APPROVED** [review]

File: src/pages/steps/CalendarStep/CalendarStep.tsx [hunk 3]
```diff
@@ -178,8 +192,9 @@ const CalendarStep = <TFormData extends FormDataUnion>({
         ? calendarStateType
         : Object.keys(calendarStateType)[0];
 
+    console.log('GOING TO MUTATE');
```
Comment (brampauwelyn) [inline src/pages/steps/CalendarStep/CalendarStep.tsx:195]: Remove console.log

File: src/types/Place.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import type { Address } from './Address';
-import type { Offer } from './Offer';
+import type { InternalOffer } from './Offer';
```
Comment (brampauwelyn) [inline src/types/Place.ts:2]: Maybe we could rename it to `BaseOffer` ?

**Pull request merged**